runtime/runtime-core/common/src/aws/smithy/kotlin/runtime/retries/StandardRetryStrategy.kt

```diff
@@ -131,25 +168,121 @@ public class StandardRetryStrategy(
     private fun <R> throwTooManyAttempts(attempt: Int, result: Result<R>): Nothing =
         when (val ex = result.exceptionOrNull()) {
             null -> throw TooManyAttemptsException(
-                "Took more than ${options.maxAttempts} to get a successful response",
+                "Took more than ${config.maxAttempts} to get a successful response",
```

Comment:
**nit**: "attempts" is missing from this exception message:
`"Took more than ${config.maxAttempts} attempts to get a successful response",`